plugin/restore/mysql/mysql.go

```diff
@@ -123,6 +132,170 @@ func GetPITRDatabaseName(database string, timestamp int64) string {
 	return getSafeName(database, suffix)
 }
 
+// SyncBinlogs syncs the binlogs between the instance and `saveDir`, but exclude latest binlog
+func (r *Restore) SyncBinlogs(ctx context.Context, instance *api.Instance, saveDir string) error {
```

Comment:
Binlog usually conceptually refers to the MySQL binlog.
I prefer `SyncBinlogFiles` here.